pages/shoppingCart.js

```diff
@@ -17,35 +19,69 @@ export default function ShoppingCart({ ticket }) {
     }
   }, []);
 
+  const handleDeleteTicket = (id) => {
```

Comment:
You could name the `id` by what it actually is, namely an `index`. If you already have the same name for some other index, use abbreviations like `idx`.